Tools/WebKitTestRunner/libwpe/EventSenderProxyLibWPE.cpp

```diff
@@ -333,12 +333,40 @@ void EventSenderProxy::keyDown(WKStringRef keyRef, WKEventModifiers wkModifiers,
 #endif
 }
 
-void EventSenderProxy::rawKeyDown(WKStringRef key, WKEventModifiers modifiers, unsigned keyLocation)
+void EventSenderProxy::rawKeyDown(WKStringRef key, WKEventModifiers wkModifiers, unsigned keyLocation)
```

Comment:
/* Unofficial Review */
This rawKeyDown and rawKeyUp below are almost duplicated except for two booleans. Can we put their code together into a single method with an `isKeyPressed` boolean argument as we do in EventSenderProxyGtk.cpp?